src/com/xilinx/rapidwright/edif/EDIFCell.java

```diff
@@ -62,6 +63,14 @@ public class EDIFCell extends EDIFPropertyObject {
 
     private EDIFName view = DEFAULT_VIEW;
 
+    /**
+     * An atomically updated variable to track the number of `EDIFCellInst`
+     * objects (attached to a parent cell) that instantiate this cell.
+     */
+    private volatile int instanceCount = 0;
```

Comment:
`directInstanceCount`?
`explicitInstanceCount`?
`foldedInstanceCount`?

Reply:
I've gone with `nonHierInstantiationCount` to make it clear that it's most definitely not `Hier`.